config-model/src/main/java/com/yahoo/vespa/model/container/search/ContainerSearch.java

```diff
@@ -59,6 +59,7 @@ private void initializeDispatchers(Collection<AbstractSearchCluster> searchClust
         for (AbstractSearchCluster searchCluster : searchClusters) {
             if ( ! ( searchCluster instanceof IndexedSearchCluster)) continue;
             owningCluster.addComponent(new DispatcherComponent((IndexedSearchCluster)searchCluster));
+            owningCluster.addComponent(new RpcResourcePoolComponent((IndexedSearchCluster)searchCluster));
```

Comment:
/source/config-model/src/main/java/com/yahoo/vespa/model/container/search/ContainerSearch.java:[62,44] cannot find symbol
  symbol:   class RpcResourcePoolComponent
  location: class com.yahoo.vespa.model.container.search.ContainerSearch
[INFO] 1 error

I guess you mean RpcResourcePool

Reply:
No the class name is correct, but I forgot to do 'git add'.